atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServices.java

```diff
@@ -92,42 +104,40 @@ static void waitForSchemaVersions(Cassandra.Client client, String tableName, int
                 .append(" and examine their logs to determine the issue.")
                 .append(" Fixing the underlying issue and restarting Cassandra")
                 .append(" should resolve the problem. You can quick-check this with 'nodetool describecluster'.");
-        throw new IllegalStateException(sb.toString());
+
+        // 1 real schema version that is correctly distributed + precisely one node that is at version 'unreachable'
+        if (allowUnresponsiveNode
+                && versions.entrySet().size() == 2
+                && versions.keySet().contains("UNREACHABLE")
+                && versions.get("UNREACHABLE").size() == 1) {
+            log.error(sb.toString());
+        } else {
+            throw new IllegalStateException(sb.toString());
+        }
     }
 
     /**
      * This is a request from pbrown / FDEs; basically it's a pain to do DB surgery to get out
      * of failed patch upgrades, the majority of which requires schema mutations; they would find
      * it preferable to stop before starting the actual patch upgrade / setting APPLYING state.
      */
-    static void failQuickInInitializationIfClusterAlreadyInInconsistentState(
+    static void warnUserInInitializationIfClusterAlreadyInInconsistentState(
             CassandraClientPool clientPool,
             CassandraKeyValueServiceConfig config) {
-        if (config.safetyDisabled()) {
-            log.error("Skipped checking the cassandra cluster during initialization, because safety checks are"
-                    + " disabled. Please re-enable safety checks when you are outside of your unusual"
-                    + " migration period.");
-            return;
-        }
         String errorMessage = "While checking the cassandra cluster during initialization, we noticed schema versions"
-                + " could not settle. Failing quickly to avoid getting into harder to fix states (i.e. partially"
-                + " applied patch upgrades, etc). This state is in rare cases the correct one to be in; for"
-                + " instance schema versions will be incapable of settling in a cluster of heterogenous"
-                + " Cassandra 1.2/2.0 nodes. If that is the case, disable safety checks in your Cassandra"
-                + " KVS preferences.";
+                + " could not settle. Be aware that some operations will not work while you are in your current"
+                + " cluster status.";
         try {
             clientPool.run(client -> {
                 waitForSchemaVersions(
                         client,
                         "(none, just an initialization check)",
-                        config.schemaMutationTimeoutMillis());
+                        config.schemaMutationTimeoutMillis(),
+                        true);
                 return null;
             });
         } catch (TException e) {
             throw new RuntimeException(errorMessage, e);
-        } catch (IllegalStateException e) {
```

Comment:
Yes, I'm not sure what the point of logging here and re-throwing is.
Nothing above this specifically does anything with the IllegalStateException and it will just get logged as part of the "everything is dead" stacktrace.